gsoc2018-bharat/src/CheckPerson.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+import sys
+
+def countPronouns(filename, f):
```

Comment:
really nice to have docstrings in the code :-) , but there are some other python style issues that would be worth complying with. Filenames are all in lowercase, e.g. `check_person.py` and function names are in snake case `count_pronouns`, snake case is only reserved for class names.

Reply:
I will update the scripts accordingly.